spec/features/document_sets_spec.rb

```diff
@@ -24,15 +24,18 @@
     @collection.hide_completed = false
     @collection.save
     #change work restrictions temporarily so they don't interfere with doc set permissions
-    work = Work.find_by(id: @set.works.first.id)
+    unless @owner.account_type == "Individual Researcher"
```

Comment:
I'd prefer not to avoid this test, but rather to make sure that `@owner` is a different user record with the appropriate account type.